chainio/txmgr/txmgr.go

```diff
@@ -15,6 +16,11 @@ import (
 	"github.com/ethereum/go-ethereum/log"
 )
 
+var (
+	FallbackGasTipCap          = big.NewInt(15_000_000_000)
```

Comment:
super high imo

Reply:
5gwei should be plenty

Reply:
[b733a74](https://github.com/Layr-Labs/eigensdk-go/pull/167/commits/b733a74ae6525e3ddbc5eaee2aa73815e900e037)